### DISCUSSION TITLE
Cell-type classification for positive units

### DISCUSSION THREAD

**Discussion by ErnestJohnston** [petersenpeter/CellExplorer#98] [Q&A] [answered]
Hello,

After reading the documentation on cell-type classification, it seems that one of the main criteria for the procedure is the trough-to-peak time of the waveform. How does this criteria apply to positive units? They do not seem to have a morphology that lends itself to trough-to-peak (or conversely, peak-to-trough) measurements in the same way that canonical negative units do.

Thank you!

**Comment by petersenpeter** [comment, marked as answer]
Hi

You have to rely on other features for positive waveforms. The firing rate, burstiness, connectivity, and the autocorrelogram. 

Where are you recording?
